nexus/src/external_api/http_entrypoints.rs

```diff
@@ -316,10 +319,10 @@ pub fn external_api() -> NexusApiDescription {
 /// Fetch the top-level IAM policy
 #[endpoint {
     method = GET,
-    path = "/policy",
+    path = "/global/policy",
```

Comment:
As @zephraph [suggested](https://github.com/oxidecomputer/omicron/issues/1571#issuecomment-1209982957), this breaks off a little chunk of the big `/global/*` change just to unblock this work. The `/global/` thing is so big that I think this approach makes sense regardless, though the downside is we have a short time where some global things have the prefix and some don't.